### PR TITLE
Describe requirements on db.logs.query.annotation_data_* configuration settings

### PR DESCRIPTION
**Pull request by alexnb**
Describe requirements on db.logs.query.annotation_data_* configuration settings.

----

If you open a PR that needs to go into a current version, you need to *cherry-pick your commit from dev over to the current version branch*. Only then will the proper builds that generate html/pdf be run. But beware: Docs will be generated but not published automatically!

- [x] N/A - or - I have added the appropriate "cherry-pick-to" labels to this PR so I don't forget to do this later!

### CODE REVIEW

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates.

You can view the updated docs at https://neo4j-docs-ops-manager-151.surge.sh

File: modules/ROOT/pages/addition/instance-requirements.adoc [hunk 1]
```diff
@@ -1,8 +1,6 @@
 = Neo4j instance requirements
 
-In order to be managed by NOM, a Neo4j instance **must**:
-
-* be running a Neo4j Enterprise Edition v4.4.0 or higher (including all 5.x versions).
+In order to be managed by NOM, a Neo4j instance **must** be running a Neo4j Enterprise Edition v4.4.0 or higher (including all 5.x versions).
```
Comment (SaschaPeukert) [inline modules/ROOT/pages/addition/instance-requirements.adoc]:
```suggestion
In order to be managed by NOM, a Neo4j instance **must** be running a Neo4j Enterprise Edition v4.4.2 or higher (including all 5.x versions).
```

Comment (SaschaPeukert) [reply modules/ROOT/pages/addition/instance-requirements.adoc]: We should have updated that ages ago. At least in the compatibility matrix it was right

**Review by SaschaPeukert — APPROVED** [review]
If you include my little change, I am very happy with this :)

**Comment by alexnb** [comment]
@AlexicaWright Hi Jessica, could you please have a look at this one and cherry pick it. Unfortunately I already merged it to `dev`.

**Pull request merged**